PSL_catalog.json

```diff
@@ -0,0 +1,156 @@
+{
+    "project_one_line": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/B-Tax",
+        "type": "html",
+        "data": "<p>Evaluates the effect of US federal taxes on businesses' investment incentives</p>"
+    },
+    "key_features": {
+        "start_header": null,
+        "end_header": null,
+        "source": null,
+        "type": null,
+        "data": null
+    },
+    "project_overview": {
+        "start_header": "B-Tax",
+        "end_header": "Disclaimer",
+        "source": "README.md",
+        "type": "github_file",
+        "data": null
+    },
+    "citation": {
+        "start_header": "Citing the B-Tax Model",
+        "end_header": null,
+        "source": "README.md",
+        "type": "github_file",
+        "data": null
+    },
+    "license": {
+        "start_header": null,
+        "end_header": null,
+        "source": "https://github.com/open-source-economics/B-Tax/blob/master/LICENSE.md",
+        "data": "<p>CC0 1.0 Universal (CC0 1.0) Public Domain Dedication</p>",
+        "type": "html"
+    },
+    "user_documentation": {
```

Comment:
@Peter-Metz You might link to this document here: https://github.com/open-source-economics/B-Tax/blob/master/Guides/CCC_Guide.pdf